Sources/TootSDK/TootClient/TootClient.swift

```diff
@@ -160,14 +160,15 @@ extension TootClient {
 
     /// Fetch data asynchronously and return the raw response.
     internal func fetch(req: HTTPRequestBuilder) async throws -> (Data, HTTPURLResponse) {
-        if req.headers.index(forKey: "Content-Type") == nil && req.body != nil {
+        if req.headers.index(forKey: "Content-Type") == nil {
```

Comment:
I think this check is also sharkey specific.

Reply:
I undid my change and moved it to a separate if clause that checks for sharkey (although I don't consider it sharkey-specific, seems to me more generally correct and works with all platforms I tested but happens to be necessary for sharkey). The files changed shows that first part is the same as before.